apps/src/studio/levels.js

```diff
@@ -2551,19 +2553,22 @@ levels.js_hoc2015_add_characters = {
     'addCharacter("PufferPig");',
     '',
     'function whenGetPufferPig() {',
-    '  playSound("PufferPigSound1");',
+    '  playSound("PufferPigRandom");',
```

Comment:
Realising it isn't new with this change, but it is odd to have both random and Random for sounds for different characters?